shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -56,6 +57,12 @@
     @Getter
     private final long dynamicCacheSize;
 
+    /**
+     * Cache eviction duration for dynamic metrics.
+     */
+    @Getter
+    private final int dynamicCacheEvictionDuration;
```

Comment:
rename this to capture the duration in the name..

Reply:
changed it to `dynamicCacheEvictionDurationMs`